lib/stanford-mods/searchworks.rb

```diff
@@ -127,12 +126,16 @@ def main_author_w_date_test
 
       # @return [String] value for title_245a_search field
       def sw_short_title
-        short_titles ? short_titles.first : nil
+        short_titles ? short_titles.compact.reject(&:empty?).first : nil
       end
 
+      # @return [Nokogiri::XML::NodeSet] title_info nodes, rejecting ones that just have blank text values
+      def outer_nodes
```

Comment:
makes sense to me, i just extracted the variable name from the method it used to be in so I could apply the logic in one place, but agree it could more clear as to what is actually is